### DISCUSSION TITLE
Using Cinemagoer to enhance a list

### DISCUSSION THREAD

**Discussion by HeavyTony2** [cinemagoer/cinemagoer#491] [Q&A]
I've a list of films without countries. I like to use Cinemagoer to retrieve their countries. Is it possible to use it without having to manually query each entries of my list?

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
